uri/exid-types/GOV-ID.yaml

```diff
@@ -1,13 +1,15 @@
 %YAML 1.2
 ---
+label: GOV Historic Gazeteer ID
+
 lang: en
 
 type: uri
 
-uri: https://gedcom.io/exid-type/GOV-ID
+uri: http://gov.genealogy.net/
```

Comment:
Here I noticed that exid-types.json (from https://github.com/FamilySearch/GEDCOM/pull/566) had one URI and GOV-ID.yaml (from https://github.com/FamilySearch/GEDCOM-registries/pull/96) had a different URI.  I think the exid-types.json is the correct one, but not sure.